src/Evolution/Systems/Cce/WorldtubeBufferUpdater.cpp

```diff
@@ -454,6 +454,94 @@ void BondiWorldtubeH5BufferUpdater::pup(PUP::er& p) {
   }
 }
 
+KleinGordonWorldtubeH5BufferUpdater::KleinGordonWorldtubeH5BufferUpdater(
+    const std::string& cce_data_filename,
+    const std::optional<double> extraction_radius)
+    : cce_data_file_{cce_data_filename}, filename_{cce_data_filename} {
+  get<Tags::detail::InputDataSet<
+      Spectral::Swsh::Tags::SwshTransform<Tags::KleinGordonPsi>>>(
+      dataset_names_) = "KGPsi";
+  get<Tags::detail::InputDataSet<
+      Spectral::Swsh::Tags::SwshTransform<Tags::KleinGordonPi>>>(
+      dataset_names_) = "dtKGPsi";
+
+  const std::string text_radius = detail::get_text_radius(cce_data_filename);
+  try {
+    extraction_radius_ = static_cast<bool>(extraction_radius)
```

Comment:
I know this is how the bondi buffer updater did it, but `static_cast<bool>(extraction_radius)` can just be `extraction_radius.has_value()`